src/lib/render-lit-html.ts

```diff
@@ -181,7 +181,7 @@ export async function* renderValue(
         const templateResult = (instance.instance as any).renderLight();
         yield* renderValue(templateResult, childRenderer, renderInfo);
       }
-    } else if (value === nothing) {
+    } else if (value === nothing || value === noChange) {
```

Comment:
When would this value be `noChange`. Is it just when a user explicitly sets this value or is there another case?

Reply:
It's a value users can set like any other.